hack/app_sre_pr_check.sh

```diff
@@ -7,17 +7,18 @@ MISSING_NS="false"
 for F in $(ls deploy/sre-prometheus)
```

Comment:
This fails on `deploy/sre-prometheus/aws/`. This will always cause this script to fail with `exit 2`, is this acceptable in this circumstance?

Reply:
I would suggest something like 

```
for F in $(find ./deploy/sre-prometheus -type f -iname "*PrometheusRule.yaml"); do echo $F;done
./deploy/sre-prometheus/100-excessive-memory.PrometheusRule.yaml
./deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml
./deploy/sre-prometheus/100-configure-alertmanager-operator.PrometheusRule.yaml
./deploy/sre-prometheus/100-node-unschedulable.PrometheusRule.yaml
./deploy/sre-prometheus/100-dns-latency.PrometheusRule.yaml
./deploy/sre-prometheus/aws/100-stuck-volumes.PrometheusRule.yaml
./deploy/sre-prometheus/aws/100-ebs-iops-burstbalance.PrometheusRule.yaml
./deploy/sre-prometheus/100-sre-pruning.PrometheusRule.yaml
./deploy/sre-prometheus/100-kubeapilatencyhigh-custom.PrometheusRule.yaml
./deploy/sre-prometheus/100-router-health.PrometheusRule.yaml
./deploy/sre-prometheus/100-layered-sre-elevation.PrometheusRule.yaml
./deploy/sre-prometheus/100-sre-elevation.PrometheusRule.yaml
./deploy/sre-prometheus/100-cluster-upgrade.PrometheusRule.yaml
./deploy/sre-prometheus/100-cluster-operator.PrometheusRule.yaml
./deploy/sre-prometheus/100-telemeter-client.PrometheusRule.yaml
```